doc/api/worker_threads.md

```diff
@@ -913,6 +913,26 @@ Calling `unref()` on a worker will allow the thread to exit if this is the only
 active handle in the event system. If the worker is already `unref()`ed calling
 `unref()` again will have no effect.
 
+### `worker.performance.eventLoopUtilization([utilization1[, utilization2]])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `utilization1` {Object} The result of a previous call to
+    `eventLoopUtilization()`.
+* `utilization2` {Object} The result of a previous call to
+    `eventLoopUtilization()` prior to `utilization1`.
+* Returns {Object}
+  * `idle` {number}
+  * `active` {number}
+  * `utilization` {number}
+
+The same call as [`perf_hooks` `eventLoopUtilization()`][], except returns the
+values for the worker instance.
+
+Currently the event loop utilization of a worker instance can only be retrieved
+after the [`'online'` event][] event has fired.
```

Comment:
Add note about what happens if you call it before `online` has fired?

Reply:
```suggestion
after the [`'online'` event][] event is emitted.
```